hugo/content/citation/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: How to cite the Genome Portal and the data
+---
+
+In line with the principles of _FAIR_ and _Open Science_, we encourage the reuse and recognition of material made available on the Swedish Reference Genome Portal.
+On this page, you will find information about how to cite the portal when reusing and referencing the content. Please note that the information on the portal is
+updated continuously, therefore it is important to refer to specific versions (or to provide access dates) within citations.
+
+Below are instructions for how to cite different aspects of the Genome Portal. These include:
+
+- Citing the the Swedish Reference Genome Portal website
+
+- Citing a specific species entry in the Genome Portal
+
+- Citing the data in the Genome Portal
+
+- Citing the underlying code of the Genome Portal, including its dependency on JBrowse 2
```

Comment:
Optional, but this could be written as a table of contents with clickable links to each section below.

Reply:
Good idea! Added to board as EB-225.